src/panels/lovelace/editor/card-editor/hui-card-editor.ts

```diff
@@ -245,14 +249,15 @@ export class HuiCardEditor extends LitElement {
         this._handleUIConfigChanged(ev as UIConfigChangedEvent)
       );
 
+      this.GUImode = true;
```

Comment:
This makes sure the value is true but also makes sure the toggle is updated when switched to the editor. In some cases, I saw that the `this._cardEL.hasWarning` would still come back as true (it has a warning) when the editor card was updated because the warning hadn't been set to undefined yet by the editor